wallet-new/src/Cardano/Wallet/WalletLayer.hs

```diff
@@ -30,6 +31,7 @@ bracketKernelPassiveWallet
     :: forall m n a. (MonadIO m, MonadIO n, MonadMask n)
     => (Severity -> Text -> IO ())
     -> Keystore
+    -> MonadDBReadAdaptor IO
```

Comment:
This argument is not being used.

Reply:
Yes it is, it's used by the function that this function calls. This is basically just an alias.

Reply:
Ah yeah, you are correct.